Setup/Product.wxs

```diff
@@ -135,6 +138,312 @@
         <File Source="..\GitExtensions\bin\$(var.Configuration)\System.Runtime.InteropServices.RuntimeInformation.dll" />
       </Component>
 
+            <Component Id="Microsoft.Win32.Primitives.dll" Guid="*">
```

Comment:
😱  Do we really need all these files????

Reply:
https://github.com/gitextensions/gitextensions/pull/6005#discussion_r244496650

Reply:
Can we make them use the same version, or add a binding redirect?

Reply:
We already have auto binding redirect. I really don't see the problem.  Package references already resolve the nuget dependencies.  Jira plugin uses NString which uses a .net standard version of System.collections.immutable.  Composition did not.  
 Notice the issue.  https://github.com/gitextensions/gitextensions/pull/6010#issuecomment-450499723  https://github.com/gitextensions/gitextensions/pull/6005#discussion_r244496650

Reply:
📝 If we move to .NET Standard 2.0+ dependencies, most of these will no longer be required.

Reply:
Great.  I have no idea on how to do that correctly.

Reply:
Build log.  https://drive.google.com/file/d/1k1K__oDpvkQYWjRdKt9IaHgW_fiBznU8/view?usp=sharing

Reply:
If you are so unsure, you can

1. Run ```git clean -d -x -f```
1. Make sure .\GitExtensions\bin\release is empty
1. Run .\Setup\Prepare-Release.ps1 -oldVersion 3.0.0 -newVersion 3.0.1 -milestones '49'
1. Look at .\GitExtensions\bin\release for yourself.